src/main/resources/application.properties

```diff
@@ -124,6 +127,9 @@ cloudforms.manifest.0.vmworkloadinventory.numCpuPath=$.ManageIQ::Providers::Vmwa
 # TODO fix this JSONPath
 cloudforms.manifest.0.vmworkloadinventory.numCoresPerSocketPath=$.ManageIQ::Providers::Vmware::InfraManager.hosts[*].vms[?(@.id == {id})].num_cpu
 cloudforms.manifest.0.vmworkloadinventory.hasRDMDiskPath=$.ManageIQ::Providers::Vmware::InfraManager.hosts[*].vms[?(@.id == {id})].has_rdm_disk
+cloudforms.manifest.0.vmworkloadinventory.memoryHotAddEnabledPath=$.ManageIQ::Providers::Vmware::InfraManager.hosts[*].vms[?(@.id == {id})].memory_hot_add_enabled
```

Comment:
In version 0 there was no such fields , ¿?

Reply:
I've removed this and altered code to suit